src/main/java/appeng/capabilities/Capabilities.java

```diff
@@ -0,0 +1,48 @@
+package appeng.capabilities;
+
+
+import net.minecraft.nbt.NBTBase;
+import net.minecraft.util.EnumFacing;
+import net.minecraftforge.common.capabilities.Capability;
+import net.minecraftforge.common.capabilities.CapabilityInject;
+import net.minecraftforge.common.capabilities.CapabilityManager;
+
+import appeng.api.storage.IStorageMonitorableAccessor;
+
+
+/**
+ * Utility class that holds various capabilities, both by AE2 and other Mods.
+ */
+public class Capabilities
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a private constructor to hide the implicit public one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS1118)